orchestrationSpecs/packages/config-processor/src/RunMigrationInitializer.ts

```diff
@@ -59,7 +59,6 @@ async function main() {
     let etcdEndpoints = process.env.ETCD_ENDPOINTS;
     let etcdUser = process.env.ETCD_USER;
     let etcdPassword = process.env.ETCD_PASSWORD;
-    let prefix = process.env.PREFIX;
```

Comment:
nit: why drop this & not the others?  I've been setting up env variables because in some contexts, those can be easier to set up on containers than args (e.g. what if the prefix were in a configmap).